### PR TITLE
As a user clicking through to a search result, I want the search to be carried through

### PR DESCRIPTION
**Pull request by Sandravaphilips**
- Issue: https://github.com/laws-africa/peachjam/issues/352
- Loom: https://www.loom.com/share/30fe2bc271054ca28bda3fcc36d428c2

### CODE REVIEW

File: peachjam/js/components/FindDocuments/SearchResult.vue [hunk 1]
```diff
@@ -4,7 +4,7 @@
       class="h5"
       target="_blank"
       rel="noreferrer"
-      :href="item.expression_frbr_uri"
+      :href="`${item.expression_frbr_uri}?q=${query}`"
```
Comment (musangowope) [inline peachjam/js/components/FindDocuments/SearchResult.vue:7]: wrap `q` in `encodeURIComponent`

Comment (musangowope) [inline peachjam/js/components/FindDocuments/SearchResult.vue:7]: wrap `query` with `encodeURIComponent`

**Review by musangowope — CHANGES_REQUESTED** [review]
Great first step. The next step would be to accomodate pdf documents. You need to trigger the search only once the pdf document has loaded

**Comment by musangowope** [comment]
Taking videos of work done for prs can also be really valuable. We like to use a tool called [Loom](https://www.loom.com/)

File: peachjam/js/components/document-content.ts [hunk 2]
```diff
@@ -59,6 +59,13 @@ class DocumentContent {
       this.pdfRenderer = new PdfRenderer(root);
       this.pdfRenderer.onPreviewPanelClick = () => { this.navOffCanvas?.hide(); };
       this.pdfRenderer.onPdfLoaded = () => {
+        const searchInput: HTMLInputElement | null = this.root.querySelector('[aria-label="Search document content"]');
+        const searchForm: HTMLFormElement | null = this.root.querySelector('.doc-search__form');
+        if (searchInput) {
+          searchInput.value = 'law';
```
Comment (musangowope) [inline peachjam/js/components/document-content.ts:65]: this is a hard-coded value. I'm guessing you were using this for testing purposes. Please set `search.value` to the  `url param` `q`

**Review by musangowope — CHANGES_REQUESTED** [review]
Minor change

**Comment by musangowope** [comment]
it would also be amazing if we could immediately activate the search tab in `in-document search` if  url param `q` exists

**Comment by longhotsummer** [comment]
> it would also be amazing if we could immediately activate the search tab in `in-document search` if url param `q` exists

Yes please!

**Review by musangowope — APPROVED** [review]

**Pull request merged**